packages/example-webcomponent/src/webcomponent.js

```diff
@@ -0,0 +1,52 @@
+import { LitElement, html, css } from 'lit-element';
+
+export default class HelloComponent extends LitElement {
```

Comment:
I would probably name this file `hello-component.js` instead of `webcomponent.js` to stick with the class/filename matching scheme.